Lib/http/server.py

```diff
@@ -1033,6 +1033,17 @@ def is_python(self, path):
         head, tail = os.path.splitext(path)
         return tail.lower() in (".py", ".pyw")
 
+    def make_cmdline( self, scriptfile, query ):
+        cmdline = [scriptfile]
+        if self.is_python(scriptfile):
+            interp = sys.executable
+            if interp.lower().endswith("w.exe"):
+                # On Windows, use python.exe, not pythonw.exe
+                interp = interp[:-5] + interp[-4:]
```

Comment:
This check isn't necessary. The child process is spawned using pipes for standard I/O, which works fine with the non-console "pythonw.exe" executable. This is probably left over from when it called `os.popen2()` ([link](https://github.com/python/cpython/commit/e7d6b0a22e07adfa33cfa8658acf4c0b3cbecb39#diff-590102ca60c804a5ea71a9a5e03b0ae0b5806be5f2452e2ca726cdeb6092558fR216-R234) to Guido's revision that added this code 21 years ago). 

Even if it did matter, it should only check for exactly "pythonw.exe" nowadays, e.g. `os.path.normcase(os.path.basename(interp)) == os.path.normcase("pythonw.exe")`.

Reply:
I have made the change in local pushed to Github but I cant figure out how to push it to Python/cpython from my copy, can you give me a clue?